cmd/uplink/cmd/mb.go

```diff
@@ -41,19 +43,27 @@ func makeBucket(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Nested buckets not supported, use format sj://bucket/")
 	}
 
-	metainfo, _, err := cfg.Metainfo(ctx)
+	project, err := cfg.GetProject(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("error setting up project: %+v", err)
 	}
+	defer func() {
+		err = project.Close()
```

Comment:
you need to decide if you are using `err = project.Close()` or `err := project.Close()`. With first you might also return error to command line + print your message and we will have duplicated output. You have several such places.